helm-charts/secrets-operator/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Did you mean to include this file?

Reply:
It is apart of the new helmify that was auto generated. It used to exist but now they made it standalone